### PR TITLE
Select H/W version via  hw-latest feature 

### PR DESCRIPTION
**Pull request by rusty1968**
Each crate that has caliptra-drivers and/or caliptra-registers as a dependency must expose the hw-latest feature.

### CODE REVIEW

File: builder/src/firmware.rs [hunk 1]
```diff
@@ -15,48 +15,118 @@ pub fn rom_from_env() -> &'static FwId<'static> {
...
     crate_name: "caliptra-fmc",
     bin_name: "caliptra-fmc",
     features: &["emu"],
 };
 
+#[cfg(feature = "hw-latest")]
+pub const FMC_FAKE_WITH_UART: FwId = FwId {
+    crate_name: "caliptra-fmc",
+    bin_name: "caliptra-fmc",
+    features: &["emu", "fake-fmc", "hw-latest"],
```
Comment (korran) [inline builder/src/firmware.rs:78]: Seeing as we're doing this to pretty much every target I wonder if it we be more maintainable to revert the changes to this file and instead decide to add the hw-latest feature to every target in build_firmware_elfs_uncached().

Comment (rusty1968) [reply builder/src/firmware.rs:78]: Ok.  I will make the change.

Comment (rusty1968) [reply builder/src/firmware.rs]: Done.

**Review by jlmahowa-amd — APPROVED** [review]

File: runtime/log.txt [hunk 2]
```diff
@@ -0,0 +1,7749 @@
+   Compiling caliptra-runtime v0.1.0 (/home/ferrite/rusty1968/rev11-support/runtime)
```
Comment (korran) [inline runtime/log.txt:1]: Delete this file?

File: runtime/log.txt
Comment (korran) [inline runtime/log.txt:1]: Remove this?

File: runtime/log.txt (hunk 2 above)
Comment (rusty1968) [reply runtime/log.txt]: done. left-over from test.

File: runtime/log.txt
Comment (rusty1968) [reply runtime/log.txt:1]: done.

**Review by korran — APPROVED** [review]

**Review by FerralCoder — APPROVED** [review]

**Review by jhand2 — APPROVED** [review]
